assets/mascot3.original.png.license

```diff
@@ -0,0 +1,5 @@
+SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+
+SPDX-License-Identifier: AGPL-3.0-only
+
+Info: This image is based on a generated image by Midjourney (Basic subscription).
```

Comment:
Add at here for all images: Time of creation, link to midjourney terms of use/license at time of creation and add a backup of the terms/license to this repo. Just for documentation purposes down the line.